### DISCUSSION TITLE
imdb rating lookup

### DISCUSSION THREAD

**Discussion by blixten85** [autobrr/autobrr#1495] [Ideas]
I would like to see a feature where all releases that gets a match in my filter get an extra lookup at imdb for matching scores. 
If the lookup is done directly to imdb then there is no need to wait for trackers to implement ratings on their announces. 

It would be so nice to filter releases based on imdb scores along with year, genre resolution etc. I really feel that imdb lookup is missing in this tool! 😄

**Comment by nzec** [comment]
Absolutely agree. This would be a great addition if we are able to set a filter to only download movies rated higher than x or has current popularity > y.

Would allow to auto download movies which are likely to see lots of activity and are probably movies we ourselves would want to watch.

**Comment by blixten85** [comment]
Maybe the budget could be parsed too? Then I can have auto download on everything with a higher budget of $100M 😏
